docs/topics/db/queries.txt

```diff
@@ -865,6 +865,15 @@ precede the definition of any keyword arguments. For example::
 
 ... would not be valid.
 
+Differently from :meth:`~django.db.models.query.QuerySet.get` method, the 
```

Comment:
Is that really the case? I've seen a lot of cases where `get_or_create()` is used without `filter()`.

Reply:
I misunderstood this paragraph, I think though it's written in a confusing way. Actually I don't think this documentation belongs under `Q`. It should be under `get_or_create` docs. The end user would start by saying "I wish to use get_or_create(), but I can't pass it a Q obj as an arg. What should I do? I will look at get_or_create() docs." Another note that it's not about using `filter` method, it's about applying `get_or_create` to a filtered queryset, which may be obtained with a filter, exclude, etc, with or without a `Q` object.